cli/bootnode/boot_node.go

```diff
@@ -41,11 +43,15 @@ var StartBootNodeCmd = &cobra.Command{
 				MaxBackups: cfg.LogFileBackups,
 			},
 		)
+
 		if err != nil {
 			log.Fatal(err)
 		}
 
 		logger := zap.L()
+		log.Printf("starting SSV node (version %s)", commons.GetBuildData())
+
+		logger.Info(fmt.Sprintf("starting SSV %v (version %s)", cmd.Short, commons.GetBuildData()))
```

Comment:
The `AppName` is `SSV-Node`, so this seems to be printed as `starting SSV SSV-Node`, should we remove the "SSV" duplication?

Reply:
agree.

Reply:
@y0sher  Take a look at this: https://blox-ensemble.monday.com/boards/4755515647/pulses/5536501995
It seems commons.GetBuildData returns "SSV node:' ' " with no data in it.
Regardless of the bug I believe a quick validation to make sure that the data is not empty is in place here